packages/tinacms/src/admin/index.tsx

```diff
@@ -78,7 +78,7 @@ export const TinaAdmin = () => {
                         element={<CollectionCreatePage />}
                       />
                       <Route
-                        path="collections/:collectionName/:filename"
+                        path="collections/:collectionName/*"
```

Comment:
In previous version of `react-router`, you could create wildcard routes like this:
```
path="collections/:collectionName/:filename+"
```
where everything at the end of the match would be associated with `filename`.

In `react-router` v6, that does not appear to work anymore.  Instead, you can use `*` to capture "everything else" into an object:
```
const {collectionName, ...rest} = useParams()
```